tests/providers/google/cloud/operators/test_dataplex.py

```diff
@@ -243,3 +256,313 @@ def test_execute(self, lake_mock, hook_mock):
             timeout=None,
             metadata=(),
         )
+
+
+class TestDataplexRunDataQualityOperator:
+    @mock.patch(HOOK_STR)
+    @mock.patch("airflow.providers.google.cloud.operators.dataplex.DataScanJob")
```

Comment:
What about moving this string into a constant just like `HOOK_STR`?
```
"airflow.providers.google.cloud.operators.dataplex.DataScanJob"
```
It's just a style suggestion, i think it would be more consistent, but I'm not insisting.

Reply:
changed